salt/preflight/defaults.yaml

```diff
@@ -0,0 +1,27 @@
+mandatory:
+  packages:
+    - util-linux            # provides nsenter, mount
+    - iproute               # provides ip
+    - iptables              # provides iptables
+  sysctl_values:
+    net.bridge.bridge-nf-call-ip6tables: 1
+    net.bridge.bridge-nf-call-iptables: 1
+    net.ipv4.ip_forward: 1
+  available_pathdirs:
+    - /var/lib/etcd
+    - /etc/kubernetes/manifest
+  services:
+    running:
+      - containerd
+recommended:
+  packages:
+    - ebtables              # provides ebtables
+    - ethtool               # provides ethtool
+    - socat                 # provides socat
+    - iproute               # provides tc
+    - coreutils             # provides touch
+  services:
+    running:
+      - kubelet
+    dead:
+      - firewalld
```

Comment:
No. This is where we will diverge from `kubeadm`, see https://github.com/scality/metalk8s/blob/development/2.0/docs/requirements.rst#zero-trust-networking-host-firewall